src/modules/ticket-fields/index.tsx

```diff
@@ -0,0 +1,137 @@
+import type { Field } from "./data-types/Field";
+import { Checkbox } from "./fields/Checkbox";
+import { CreditCard } from "./fields/CreditCard";
+import { DatePicker } from "./fields/DatePicker";
+import { DropDown } from "./fields/DropDown";
+import { Input } from "./fields/Input";
+import { LookupField } from "./fields/LookupField";
+import { MultiSelect } from "./fields/MultiSelect";
+import { Tagger } from "./fields/Tagger";
+import { TextArea } from "./fields/textarea/TextArea";
+
+interface TicketFieldProps {
+  field: Field;
+  baseLocale: string;
+  hasAtMentions: boolean;
+  userRole: string;
+  userId: number;
+  defaultOrganizationId: string | null;
+  organizationField?: Field | null;
+  brandId: number;
+  dueDateField?: Field;
+  handleDueDateChange?: (value: string) => void;
```

Comment:
If we are going to support the "Type" field, we will need to support the due date field as well and not mark these props as optional.
I would say that this use case has a lower priority so I'll create a Jira task for that and we can keep this caveat for now

Reply:
Reasoning again about ticket type and priority, there are a lot of things we need to take into account so I would not support them for now, but we can still keep them in the `ticket-fields` module.

Reply:
Ok so leaving it now as it is for now, later we can think which fields we will add.